R/lint.R

```diff
@@ -24,6 +24,9 @@ NULL
 #' character string, store the cache in this directory.
 #' @param ... additional arguments passed to \code{\link{exclude}}.
 #' @param parse_settings whether to try and parse the settings.
+#' @param text the string or lines to be used as if linting \code{filename}.
```

Comment:
This reads a bit strange to me... how about

> Optional argument for supplying a string or lines or lines directly, e.g. if the file is already in memory or linting is being done _ad hoc_. If `text` is supplied...

Reply:
Thanks, it is more clear in this way.